README.md

```diff
@@ -34,3 +34,32 @@ To stop and remove the containers you can use the following command:
 ```sh
 docker-compose down
 ```
+
+## setup deployment with automatic updates
+
+The containers are updated using watchtower configured in the docker-compose file. To update the docker-compose we use a systemctl timer.
```

Comment:
Why not a Cron job?

Reply:
Why not systemctl? Both accomplish the same result. I personally prefer the stystemctl solution, because it allows simpler enable and disable of the timer and you automatically have a log of the runs.

Reply:
I wouldn't have thought of using systemctl, but I think you are right. It's the better option.